pom.xml

```diff
@@ -52,6 +52,18 @@
     </developer>
   </developers>
 
+  <dependencyManagement>
```

Comment:
Why are you adding `<dependencyManagement>`? Can you remove it and only use `<dependencies>`?

Reply:
Added the change